aips/aip-50.md

```diff
@@ -0,0 +1,63 @@
+---
+aip: 50
+title: Allow direct commission change vesting contract
+author: junkil-park, michelle-aptos, movekevin, wintertoro
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/249
+Status: In Review
+type: Standard Framework
+created: 09/28/2023
+---
+
+# AIP-50 - Change commission rates in delegation pools
+
+## Summary
+Currently, the commission rate is set by the pool owner at the initiation of the pool and fixed afterward. We propose allowing the pool owner to change the commission rate after delegation pool creation.
+
+### Goals
+This feature will increase the flexibility of pool owners over network/commission rates. Pool owners can adjust operator commission rates as appropriate, based on market conditions and commercial discussions. As such, this will facilitate staking pools to participate in market dynamics. I.e. the pool can be set at a lower commission to attract more stakers, or raise commission when needed. Operators can call the function themselves if they are also the pool owner, or if not, work with the pool owner they are operating for.
+
+### Out of Scope
+Operators will not be able to change the commission rate. In our existing design setup, operators are designated by pool owners.
+
+### Impact
+* *Delegation pool owners* have permission to change the operator commission rate.
+* *Operators* are impacted by commission rate changes, which will impact their rewards.
+* *Delegators* need to watch over commission rate changes so that they are not overcharged. They may assume that the commission rate is fixed.
+
+## Alternative solutions
+There is no alternative solution because the delegation pool module currently does not allow changing the operator commission rate.
+
+Moreover, we decided against allowing operators to change the commission rate, as it would not make sense in the context of our existing design setup. Right now, Operators are designated by pool owners, so eventually, pool owners control everything.
+
+## Specification
+* The maximum commission rate is uncapped (i.e., 100%). The minimum commission rate is 0%.
```

Comment:
when we say uncapped, it does have a max which is 100%, let's not say uncap here so people might think it can be 120% :D

Reply:
Fixed. Good point!